### DISCUSSION TITLE
Does tRPC support NextJS On-demand Revalidation?

### DISCUSSION THREAD

**Discussion by arashi-dev** [trpc/trpc#2305] [Q&A] [answered]
Does tRPC support the `NextJS` [On-demand Revalidation](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration#on-demand-revalidation) method which was added since version `12.2`? or any other built-in apis?

**Comment by KATT** [comment, marked as answer]
Yeah, I think you can just call that in a mutation or whatnot as long as you propagate `res` to the ctx

Cc @gt-codes, I think you got this working?

**Reply by arashi-dev** [reply]
sounds great! thank you for your response.
